keyboards/b_sides/rev41lp/rules.mk

```diff
@@ -0,0 +1,18 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = qmk-dfu
```

Comment:
Are you sure about this? I don't see where it is configured.

Reply:
I'm not, this comes from the import of original code so I didn't touch it. I've only built firmware with catalina bootloader for pro micro.

Reply:
If it's Pro Micro compatible, then it should probably be:
```suggestion
BOOTLOADER = caterina
```

Reply:
It is, I have two built with pro micro!